packages/quantic/force-app/main/default/lwc/quanticTabBar/quanticTabBar.js

```diff
@@ -0,0 +1,361 @@
+import {LightningElement} from 'lwc';
+
+import more from '@salesforce/label/c.quantic_More';
+
+/**
+ *  The `QuanticTabBar` component displays the Quantic Tabs in a responsive manner by showing a drop-down list that will display overflowing tabs on smaller screens.
+ * @category Search
+ * @example
+ * <c-quantic-tab-bar>
+ *   <c-quantic-tab engine-id={engineId} label="Tab 1" expression={expressionOne} is-active></c-quantic-tab>
+ *   <c-quantic-tab engine-id={engineId} label="Tab 2" expression={expressionTwo}></c-quantic-tab>
+ *   <c-quantic-tab engine-id={engineId} label="Tab 3" expression={expressionThree}></c-quantic-tab>
+ * </quantic-tab-bar>
+ */
+export default class QuanticTabBar extends LightningElement {
+  labels = {
+    more,
+  };
+
+  /** @type {boolean} */
+  hasRendered = false;
+  /** @type {boolean} */
+  isComboboxOpen = false;
+  /** @type {Array<{value: string, label: string}>} */
+  options = [];
+  /** @type {number} */
+  maxMoreButtonWidth = 0;
+  /** @type {boolean} */
+  expandedMoreButton = true;
+
+  connectedCallback() {
+    window.addEventListener('resize', this.updateTabsDisplay);
+    this.addEventListener('tab_rendered', this.updateTabsDisplay);
+  }
+
+  renderedCallback() {
+    if (!this.hasRendered) {
+      this.hideMoreButton();
+      this.hasRendered = true;
+    }
+  }
+
+  /**
+   * Updates the display of the tabs.
+   * @returns {void}
+   */
+  updateTabsDisplay = () => {
+    if (this.isOverflow) {
+      this.showMoreButton();
+    } else {
+      this.hideMoreButton();
+    }
```

Comment:
I guess you could merge `showMoreButton` and `hideMoreButton` in a single function `updateMoreButtonVisibility(show: boolean)`. It would make the code a little simpler.

```suggestion
    this.updateMoreButtonVisibility(this.isOverflow);
```